src/database.rs

```diff
@@ -54,6 +54,12 @@ pub trait Database: Send + AsDynDatabase + Any + ZalsaDatabase {
         )
     }
 
+    fn unwind_if_revision_cancelled(&self) {
```

Comment:
Would be good to lift the comment from `ZalsaLocal` on here

Reply:
done!